<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-OSR87BJ4" failures="1" tests="11" name="tests.testcase1" time="105.495" errors="1" timestamp="2025-06-19T17:04:21 IST" skipped="3">
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForRelianceCom" time="6.827">
    <skipped/>
  </testcase> <!-- verifyTheValueofHighAndLowInLast52WeeksForRelianceCom -->
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForStockTataMotors" time="15.493"/>
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForRelianceCom" time="14.804"/>
  <testcase classname="tests.testcase1" name="afterMethod" time="7.858">
    <error message="class com.aventstack.extentreports.ExtentTest cannot be cast to class com.aventstack.extentreports.markuputils.Markup (com.aventstack.extentreports.ExtentTest and com.aventstack.extentreports.markuputils.Markup are in unnamed module of loader &#039;app&#039;)" type="java.lang.ClassCastException">
      <![CDATA[java.lang.ClassCastException: class com.aventstack.extentreports.ExtentTest cannot be cast to class com.aventstack.extentreports.markuputils.Markup (com.aventstack.extentreports.ExtentTest and com.aventstack.extentreports.markuputils.Markup are in unnamed module of loader 'app')
at base.BaseTest.afterMethod(BaseTest.java:82)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:836)
at org.testng.internal.invokers.TestInvoker.runAfterConfigurations(TestInvoker.java:805)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:781)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- afterMethod -->
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForAnyStocksOfNiftyFifty" time="13.066"/>
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForRelianceCom" time="20.662"/>
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForStockTataMotors" time="13.674"/>
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForAnyStocksOfNiftyFifty" time="11.923"/>
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForAnyStocksOfNiftyFifty" time="0.649">
    <skipped/>
  </testcase> <!-- verifyTheValueofHighAndLowInLast52WeeksForAnyStocksOfNiftyFifty -->
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForAnyStocksOfNiftyFifty" time="0.539">
    <failure message="expected [https://www.nseindia.com/index-tracker/NIFTY%2050] but found [about:blank]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://www.nseindia.com/index-tracker/NIFTY%2050] but found [about:blank]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at pageEvents.NiftyFiftyPageEvents.verifyNiftyFiftyUrl(NiftyFiftyPageEvents.java:28)
at tests.testcase1.verifyTheValueofHighAndLowInLast52WeeksForAnyStocksOfNiftyFifty(testcase1.java:72)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyTheValueofHighAndLowInLast52WeeksForAnyStocksOfNiftyFifty -->
  <testcase classname="tests.testcase1" name="verifyTheValueofHighAndLowInLast52WeeksForStockTataMotors" time="0.000">
    <skipped/>
  </testcase> <!-- verifyTheValueofHighAndLowInLast52WeeksForStockTataMotors -->
</testsuite> <!-- tests.testcase1 -->
